### DISCUSSION TITLE
填入帐号之后提问会一直跳人机验证

### DISCUSSION THREAD

**Discussion by A8Cl233395** [Harry-zklcdc/go-proxy-bingai#340] [Q&A]
如题，聊天会跳人机验证，在右上角画图不会，不登录直接聊天可以

**Comment by Harry-zklcdc** [comment]
这个我也没搞懂机制到底是什么样的，有时候能过，有时候不行。我自己的账号现在也很少跳验证了，基本上都是几天才有一次，可能验证过多了，风控等级降低了，不怎么跳验证了
